lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -173,6 +173,11 @@ public function walkComparison(Comparison $comparison)
                     return false !== strpos(ClosureExpressionVisitor::getObjectFieldValue($object, $field), $value);
                 };
 
+            case Comparison::ICONTAINS:
+                return function ($object) use ($field, $value) {
+                    return false !== strpos(mb_strtolower(ClosureExpressionVisitor::getObjectFieldValue($object, $field)), mb_strtolower($value));
```

Comment:
Why not using [`mb_stripos()`](http://php.net/manual/en/function.mb-stripos.php)?